docs/cluster-management/configure-api.md

```diff
@@ -49,6 +49,26 @@ auth:
         - batman
 ```
 
+### Bookend Plugins
+
+You can globally configure which built-in bookend plugins will be used during a build. By default, `sd-scm` is enabled to begin builds with a SCM checkout command.
+
+If you're looking to include a custom bookend in the API, please refer to the *Using a custom bookend* topic in the *Extending the Docker container* section.
+
+| Key | Default| Description |
+|:----|:-------|:------------|
+| BOOKENDS_SETUP | None | The ordered list of plugins to execute at the beginning of every build. Take the forms of `'["first", "second", ...]'` |
+| BOOKENDS_TEARDOWN | None | The ordered list of plugins to execute at the end of every build. Take the forms of `'["first", "second", ...]'` |
+
+
+```yaml
+# config/local.yaml
+ecosystem:
```

Comment:
I'm changing this to `bookends` here https://github.com/screwdriver-cd/screwdriver/pull/444